app/frontend/components/Score/index.js

```diff
@@ -46,6 +48,21 @@ export default class Score extends Component {
     this.props.handleSetState({ editorState }, false)
     this.props.setInputText(editorState.getCurrentContent().getPlainText(), false)
   }
+  handleClickToMidi = () => {
+    const { title, bpm } = this.props
+    const { score } = this.state
+
+    if (!score) return
+
+    const buffer = scoreToMidi(score, { tempo: bpm })
+    const url = URL.createObjectURL(new Blob([buffer], { type: 'audio/midi'}))
+    const link = document.createElement('a')
+    link.href = url
+    link.download = (title == "" ? 'rechord' : title) + '.midi'
```

Comment:
```suggestion
    link.download = `${title || 'rechord'}.mid`
```
* javascript の空文字は false なので三項演算子は不要
* MIDI の標準的な拡張子は `.mid` （たぶん（きっと
なのでこう書けるかと思います！